pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -1088,17 +1113,17 @@ func removeCryptoKeyVersion(kmsKey string) string {
 	return kmsKey
 }
 
-// encodeDiskTags encodes requested volume tags into JSON string, as GCE does
-// not support tags on GCE PDs and we use Description field as fallback.
-func encodeDiskTags(tags map[string]string) (string, error) {
+// encodeTags encodes requested volume or snapshot tags into JSON string, as GCE does
```

Comment:
This comment is a little out of date, as PD does support labels. To reduce confusion, could you reword this something like the following?

```
enocodeTags encodes requested volume or snapshot tags into a JSON string, suitable for putting into 
the PD Description field.
```

Reply:
You're right. I have updated the comment. 

I think using labels would be more convenient than using the description field (for cost monitoring for example) but even though Snapshot and PD now support labels, the constraint on label values (max length 63 char) make it unsuitable.

Reply:
Thanks. Let's wait for the tests to pass, squash your commits, and I'll lgtm.

Reply:
I have just squashed commits